### DISCUSSION TITLE
Ui MapControls position & style

### DISCUSSION THREAD

**Discussion by vadFit** [heremaps/harp.gl#2021] [Q&A] [answered]
HAPPY NEW YEAR 2021!!🌟🍾🎉
Success for your projects  &  joy to you, your family & people you love

I would like to only display  (+ : -) Horizontally on the map Canvas

This is my actual code:
<img width="598" alt="Capture d’écran 2021-01-02 à 15 30 30" src="https://user-images.githubusercontent.com/73563799/103459551-a3ed8580-4d10-11eb-8adb-4a8e502acd57.png">

How to just display (+:-) horizontally without other options (3D & Boussole) ?
In fact chnage style of  this class =>> "harp-gl_v-align"

**Comment by FraukeF** [comment, marked as answer]
Hi @PanetoneCake, 

Nice to see you back in the new year, I also wish you a very happy new year and all the best.

Regarding the MapControlsUI, ... its not really meant to be modified, but more some helper to get started with whats there, so I see two issues here, that the UI is shown on another canvas is definitly on our side, and needs fixing, I ll create an internal task for it.

For the controls to be aligned horizontally, I would consider it application dependent, and recommend you to skip the usage of MapControlUI and  bind the events on your controls directly to MapControls.setZoomLevel (https://github.com/heremaps/harp.gl/blob/master/%40here/harp-map-controls/lib/MapControls.ts#L428) .. then you are flexible to have any UI you need.... you can see in MapControlsUI (https://github.com/heremaps/harp.gl/blob/master/%40here/harp-map-controls/lib/MapControlsUI.ts#L150) that its basically doing the same.
